src/components/TextField.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { Typography, TextField } from '@material-ui/core';
+import Container from '@material-ui/core/Container';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
```

Comment:
same comments here as in `Records.tsx` about margins/padding that counteract each other. Remove unused classes.